orchestrator-swarm/src/main/java/com/sequenceiq/cloudbreak/orchestrator/swarm/DockerClientUtil.java

```diff
@@ -54,6 +61,19 @@ public static String createContainer(DockerClient client, CreateContainerCmd cre
         return response;
     }
 
+    public static void removeContainerIfExist(DockerClient client, String name) throws Exception {
+        try {
+            InspectContainerResponse inspectResponse = client.inspectContainerCmd(name).exec();
+            if (inspectResponse != null && inspectResponse.getId() != null) {
+                RemoveContainerCmd removeContainerCmd = client.removeContainerCmd(inspectResponse.getId()).withForce(true);
+                removeContainerCmd.exec();
+                return;
+            }
+        } catch (NotFoundException ex) {
+            return;
+        }
+    }
```

Comment:
please log something to be able check later if this happened and the container had to be removed